docs/snippets/web-components/button-group-story.js.mdx

```diff
@@ -12,16 +12,15 @@ export default {
    * to learn how to generate automatic titles
    */
   title: 'ButtonGroup',
+  /*
+   *👇 Render functions are a framework specific feature to allow you control on how the component renders.
+   * See https://storybook.js.org/docs/7.0/web-components/api/csf
+   * to learn how to use render functions.
+   */
+  render: (args) => ButtonGroup(args),
```

Comment:
It seems like we should move the `render` function into the Meta whenever it makes sense to do so, as that will match the majority of the users' experience. Agreed?

Reply:
Note that I have _not_ applied this change to all of the files modified/created here, until we agree on the way forward.

Reply:
Yes, we should, as it's the default experience when a user adds Storybook to their Web Component's project. I've already been applying it in my pull request.